doc/tigrc.5.adoc

```diff
@@ -179,9 +179,12 @@ The following variables can be set:
 
 	A space-separated string of format strings used for formatting reference
 	names. Wrap the name of the reference type with the characters you would
-	like to use for formatting, e.g. `[tag]` and `<remote>`. If no format is
-	specified for `local-tag`, the format for `tag` is used. Similarly, if no
-	format is specified for `tracked-remote` the `remote` format is used.
+	like to use for formatting, e.g. `[tag]` and `<remote>`.
+	- If no format is specified for `local-tag`, the format for `tag` is used.
+	- If no format is specified for `tracked-remote`, the format for `remote`
+	  is used.
+	- If no format is specified for any other reference type, the format for
+	  `branch` is used.
```

Comment:
This breaks the layout of the generated man page. Separating list in asciidoc while staying in the same block doesn't look straightforward.

Reply:
I can't get it to work with the legacy https://pypi.org/project/asciidoc/ either, but asciidoctor can handle it - switching the man pages to that is probably a more complicated endeavour though. I've reverted it back to the running text style.